jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -87,18 +98,35 @@ pipeline {
                         }
                     }
                 }
-                stage('build-x64') {
+                stage('build-and-test-x64') {
                     agent {
                         docker {
-                            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+                            label AGENT_X64
                             image dockerAgent.image
                             args dockerAgent.args
                             alwaysPull true
                         }
                     }
                     steps {
                         script {
-                            buildAssembleUpload()
+                            def buildManifestObj = buildAssembleUpload(
+                                inputManifest: "manifests/${INPUT_MANIFEST}"
+                            )
+                            String buildManifestUrl = buildManifestObj.getUrl(JOB_NAME, BUILD_NUMBER)
+
+                            echo "buildManifestUrl (x64): ${buildManifestUrl}"
+
+                            String testManifest = "manifests/${TEST_MANIFEST}"
+                            if (fileExists(testManifest)) {
```

Comment:
@dblock @peternied any other suggestion for how to handle this case? For example, plugins typically aren't on the next major version (e.g. 2.0.0), and there is no test manifest located in `manifests/2.0.0`, and I'm assuming it's for this reason.

This solution just skips running the tests and adds some output explaining that no tests were ran since no test manifest was found.

Reply:
What about not bothering to check? If the integration tests workflow can fail because the config doesn't exist seems appropriate to me.

We avoid having problems with build manifest not existing by having them automatically generated on new version detection [GitHub action](https://github.com/opensearch-project/opensearch-build/blob/main/.github/workflows/versions.yml), [Manifest workflow](https://github.com/opensearch-project/opensearch-build/tree/main/src/manifests_workflow).  we should update that logic to add a test manifest automatically.  Alternatively this might be an appropriate time to merge the input manifest and test manifests into one

Reply:
I see, so the reason there's no test manifest for 2.0.0 is because of the manual burden. And there really should be a test manifest for every input manifest. In that case I agree it makes sense to let the integ test workflow fail if no test manifest is passed.

Regarding automated test manifest generation, I'll look into it as a future PR.